scripts/rowbind_tsv.R

```diff
@@ -0,0 +1,97 @@
+###############################################################################
+
+# pkgs require for running the script
+general_pkgs <- c("here", "magrittr", "optparse", "yaml")
+pkgs <-  c(general_pkgs, "readr", "purrr")
+
+for (pkg in pkgs) {
+  suppressPackageStartupMessages(
+    library(pkg, character.only = TRUE)
+  )
+}
+
+###############################################################################
+
+define_parser <- function() {
+  description <- paste(
+    "Collapse a set of homogeneous-structured .tsv files into a single .tsv."
+  )
+
+  parser <- OptionParser(
+    description = description
+  ) %>%
+    add_option(
+      c("--config"), type = "character",
```

Comment:
replace --config and --input_pattern with collection of filepaths (either as final positional arg, or as parsed from a single-column temp file)

Reply:
Done (the --input_files version at least)